spec/services/integrations/aggregator/taxes/invoices/void_service_spec.rb

```diff
@@ -78,6 +78,45 @@
         end
       end
 
+      context "when void invoice sync is successful for avalara integration" do
+        let(:integration) { create(:avalara_integration, organization:) }
+        let(:integration_customer) { create(:avalara_customer, integration:, customer:) }
+        let(:endpoint) { "https://api.nango.dev/v1/avalara/void_invoices" }
+        let(:body) do
+          path = Rails.root.join("spec/fixtures/integration_aggregator/taxes/invoices/success_response_void.json")
+          File.read(path)
+        end
+        let(:headers) do
+          {
+            "Connection-Id" => integration.connection_id,
+            "Authorization" => "Bearer #{ENV["NANGO_SECRET_KEY"]}",
+            "Provider-Config-Key" => "avalara-sandbox"
+          }
+        end
+        let(:params) do
+          [
+            {
+              "company_code" => integration.company_code,
+              "id" => invoice.id
+            }
+          ]
+        end
+
+        before do
+          allow(lago_client).to receive(:post_with_response).with(params, headers).and_return(response)
+          allow(response).to receive(:body).and_return(body)
+        end
+
+        it "returns invoice_id" do
+          result = service_call
+
+          aggregate_failures do
```

Comment:
`aggregate_failures` is not required as we do it by default in all tests.

Reply:
Updated